pelicun/uq.py

```diff
@@ -696,25 +683,24 @@ def _neg_log_likelihood(
     likelihoods = np.clip(likelihoods, a_min=np.nextafter(0, 1), a_max=None)
 
     # calculate the total negative log likelihood
-    NLL = -(
+    null = -(
         np.sum(np.log(likelihoods))  # from samples
         + censored_count * np.log(cen_likelihood)
     )  # censoring influence
 
     # normalize the NLL with the sample count
-    NLL = NLL / samples.size
+    return null / samples.size
 
     # print(theta[0], params, NLL)
```

Comment:
If the return statement is moved up, we should move the commented out print statement as well because it belongs to that function.

Reply:
69a867a2